src/shared/components/linkButton/linkButton.css

```diff
@@ -13,6 +14,7 @@
   width: 200px;
   cursor: pointer;
   text-decoration: none;
+  display: inline-block;
 }
```

Comment:
@kylemh Do we really want *all* LinkButton components to now have `margin: 5px` and `display: inline-block` ? This change doesn't look like it underwent review, and it will affect every part of the app where LinkButton was used.

Reply:
I definitely wanted to review it.

I did think very carefully about the decision in recommending @ksmacleod99 to add that. You aren't able to apply `width` to an inline element without setting `display` to `flex`, `inline-block`, or `block`.

In looking at the website as it stands, I haven't noticed any problems luckily. Have you?

Reply:
I haven't noticed any problems persay (only did a quick once-over) - just noticed the margin on those LinkButtons. I don't think that a shared button component should inherently *have* margin at all, though, it decreases the re-usability of it.  Now we can't use it as easily in places where there isn't room to enforce that margin.

Same goes with applying a set display style to it.

If we needed space around it in the Benefits page, there should have been a wrapper element around it for that purpose.

Reply:
Nothing wrong with the code by @ksmacleod99 here - just wanted to point out that CSS changes to a shared component shouldn't have been merged in here without a review.  This is the kind of stuff that we have to watch out for as we want to maintain the front-end of this application.

Reply:
I pm'd @davidmolina about that. I do believe changing display to utilize width was a good move, but perhaps we should look into decreasing the margin.